src/content/en/tools/workbox/guides/common-recipes.md

```diff
@@ -22,9 +22,11 @@ workbox.routing.registerRoute(
   new RegExp('https://fonts.googleapis.com/(.*)'),
   workbox.strategies.cacheFirst({
     cacheName: 'googleapis',
-    cacheExpiration: {
-      maxEntries: 30
-    }
+    plugins: [
+      new workbox.expiration.Plugin({
+        maxEntries: 30
```

Comment:
Not sure if we stick to Google style here, but if so these should have commas at the end (here and throughout).